recipe/meta.yaml

```diff
@@ -1,19 +1,18 @@
 {% set name = "cdat_info" %}
-{% set version = "2.8" %}
-#{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
-# sha256 is the prefered checksum -- you can get it for a file with:
+{% set version = "2.8.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-    git_rev: v2.8
+    git_rev: {{ version }}
     git_url: git://github.com/UV-CDAT/uvcdat.git
 
 build:
   number: 0
   skip: True  # [win]
+  noarch_python: True
```

Comment:
This means nothing for `conda-forge` (yet) and we do not recommend using it until we figure out a better way to configure the CIs.